extensions/systemd/index.js

```diff
@@ -10,6 +10,26 @@ const {Extension, errors} = require('../../lib');
 const {ProcessError, SystemError} = errors;
 
 class SystemdExtension extends Extension {
+    migrations() {
+//        const migrations = require('./migrations');
+        return [{
+            before: '1.15.0',
+            title: 'Saving node binary',
+            skip: ({instance}) => instance.process.name !== 'systemd' || instance.nodeBinaryIsKnown(),
+            task: async ({instance}) => {
+                const serviceFilename = `ghost_${instance.name}.service`;
+                const fullFilePath = path.resolve(instance.dir, './system/files', serviceFilename);
+
+                const fileContents = await fs.readFile(fullFilePath, 'utf8');
+
+                const nodeBinaryLine = fileContents.split('\n').filter(line => line.startsWith('ExecStart'))[0];
+                const nodeBinary = nodeBinaryLine.slice(nodeBinaryLine.indexOf('=') + 1, nodeBinaryLine.indexOf(' '));
+
+                instance.nodeBinary = nodeBinary;
```

Comment:
```suggestion
                const [, nodeBinary] = nodeBinaryLine.split('=');

                instance.nodeBinary = nodeBinary.trim();
```

Reply:
The line looks like this:

`ExecStart=/usr/bin/node /usr/bin/ghost run`

Would you prefer `nodeBinary.split(' ').shift()`?

Reply:
oh whoops - forgot it was put like that...

yeah in that case `nodeBinary.trim().split(' ').shift()` should work (i'd keep the `trim` just in case there's a space between the `=` and the `/usr/bin/node`